docs/download.md

```diff
@@ -31,7 +31,7 @@ The plugin is tested on the following architectures:
 
 	NVIDIA Driver*: 470+
 
-	Python 3.6+, Scala 2.12, Java 8, Java 17
+	Python 3.6+, Scala 2.12, Java 8, Java 11, Java 17
```

Comment:
We probably should only highlight what is different from Spark requirements under https://spark.apache.org/docs/{version}/ e.g., https://spark.apache.org/docs/3.4.1/

Otherwise the doc is almost always out of date with respect to the customer's version of Spark

E.g. Spark lists Python 3.7+ as a requirement since 3.3.0

Maybe 
 ```suggestion
Consult Spark documentation  for Python and Java version support at `https://spark.apache.org/docs/{version}/`. We do not support Spark builds for Scala 2.13 yet.
 ```

Reply:
Makes sense Gera. I have created another [PR9115](https://github.com/NVIDIA/spark-rapids/pull/9115) to branch-23.10 to address your suggestion. I will cherry pick the commit to this branch after PR is merged.